<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>double_p</class>
 <widget class="QWidget" name="double_p">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>331</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QGraphicsView" name="graphicsView_p13"/>
    </item>
    <item row="3" column="0">
     <widget class="QGraphicsView" name="graphicsView_p113"/>
    </item>
    <item row="2" column="0">
     <widget class="QGraphicsView" name="graphicsView_p19"/>
    </item>
    <item row="3" column="1">
     <widget class="QGraphicsView" name="graphicsView_p114"/>
    </item>
    <item row="2" column="1">
     <widget class="QGraphicsView" name="graphicsView_p110"/>
    </item>
    <item row="0" column="3">
     <widget class="QGraphicsView" name="graphicsView_p14"/>
    </item>
    <item row="3" column="2">
     <widget class="QGraphicsView" name="graphicsView_p115"/>
    </item>
    <item row="3" column="3">
     <widget class="QGraphicsView" name="graphicsView_p116"/>
    </item>
    <item row="1" column="3">
     <widget class="QGraphicsView" name="graphicsView_p18"/>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView_p12"/>
    </item>
    <item row="2" column="3">
     <widget class="QGraphicsView" name="graphicsView_p112"/>
    </item>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView_p11"/>
    </item>
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView_p15"/>
    </item>
    <item row="2" column="2">
     <widget class="QGraphicsView" name="graphicsView_p111"/>
    </item>
    <item row="1" column="1">
     <widget class="QGraphicsView" name="graphicsView_p16"/>
    </item>
    <item row="1" column="2">
     <widget class="QGraphicsView" name="graphicsView_p17"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>331</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView_p25"/>
    </item>
    <item row="2" column="0">
     <widget class="QGraphicsView" name="graphicsView_p29"/>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView_p22"/>
    </item>
    <item row="0" column="3">
     <widget class="QGraphicsView" name="graphicsView_p24"/>
    </item>
    <item row="0" column="2">
     <widget class="QGraphicsView" name="graphicsView_p23"/>
    </item>
    <item row="3" column="0">
     <widget class="QGraphicsView" name="graphicsView_p213"/>
    </item>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView_p21"/>
    </item>
    <item row="2" column="1">
     <widget class="QGraphicsView" name="graphicsView_p210"/>
    </item>
    <item row="3" column="1">
     <widget class="QGraphicsView" name="graphicsView_p214"/>
    </item>
    <item row="3" column="3">
     <widget class="QGraphicsView" name="graphicsView_p216"/>
    </item>
    <item row="2" column="3">
     <widget class="QGraphicsView" name="graphicsView_p212"/>
    </item>
    <item row="3" column="2">
     <widget class="QGraphicsView" name="graphicsView_p215"/>
    </item>
    <item row="1" column="2">
     <widget class="QGraphicsView" name="graphicsView_p27"/>
    </item>
    <item row="2" column="2">
     <widget class="QGraphicsView" name="graphicsView_p211"/>
    </item>
    <item row="1" column="3">
     <widget class="QGraphicsView" name="graphicsView_p28"/>
    </item>
    <item row="1" column="1">
     <widget class="QGraphicsView" name="graphicsView_p26"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
